packages/mobile/src/redux/reducers.ts

```diff
@@ -101,6 +103,7 @@ export interface RootState {
   walletConnect: WalletConnectState
   tokens: TokensState
   supercharge: SuperchargeState
+  [cloudFunctionsApi.reducerPath]: unknown
```

Comment:
Do you think we'll need the actual type at some point?

Reply:
I don't see why but the main reason I didn't include it is that I couldn't find it :(